README.md

```diff
@@ -1,166 +1,50 @@
+![Intercom](samples/Intercom_logo-github.png)
 # Intercom for Android
 
 ## Install
+There are 3 options for installing Intercom on your Android app.
+### Option 1: Install Intercom with Google Cloud Messaging (GCM) 
 
-Add the following dependency to your `build.gradle` file:
-
-```gradle
+Add the following dependency to your app's `build.gradle` file:
+```groovy
+dependencies {
 compile 'io.intercom.android:intercom-sdk:3.+'
```

Comment:
Should we make this
```groovy
compile 'io.intercom.android:intercom-sdk-base:3.+'
compile 'io.intercom.android:intercom-sdk-gcm:3.+'
```
now to encourage people off this and keep it consistent with the other options?